src/jwt-verifier.ts

```diff
@@ -229,7 +229,7 @@ export async function verifyJwt(
  * @param transformJwkToKeyObjectFn A function that can transform a JWK into a crypto native key object
  * @returns Promise that resolves to the payload of the JWT––if the JWT is valid, otherwise the promise rejects
  */
-async function verifyDecomposedJwt(
+export async function verifyDecomposedJwt(
```

Comment:
If we inherit from the base class like CognitoJwtVerifier does too we don't need to export these.

Composition > inheritance, but since CognitoJwtVerifier also uses inheritance, I kinda feel it makes sense to stick to that pattern (it has upsides too btw)